rpcs3/Emu/System.cpp

```diff
@@ -1682,6 +1677,17 @@ game_boot_result Emulator::Load(const std::string& title_id, bool add_only, bool
 		{
 			if (ppu_exec == elf_error::ok)
 			{
+				if (!fs::is_file(g_cfg.vfs.get_dev_flash() + "sys/external/liblv2.sprx"))
+				{
+					extern bool show_firmware_missing_message_box();
+
+					if (!show_firmware_missing_message_box())
```

Comment:
uff. just make a new emu callback to handle this in the gui_application/headless_application and spare us the buggy singleton.

Reply:
Fixed.